src/calculateRentalCost.js

```diff
@@ -28,7 +28,14 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days >= 3 && days < 7) {
+    return (days * 40) - 20;
+  } else {
+    return (days * 40) - 50;
+  }
+// write code here
```

Comment:
let's create variables for these numbers

checklist 1, 3 https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md